python/py-gitdb/Portfile

```diff
@@ -13,11 +13,11 @@ license             BSD
 description         GitDB is a pure-Python git object database
 long_description    ${description}
 
-python.versions     27 36 37
+python.versions     35 36 37
```

Comment:
please add 38 instead of 35

Reply:
when keeping `py27-gitpython` you have to keep the py37 subport here as well; also is Python 3.8 not supported (you set this conversation to resolved without changing anything).

Reply:
It's been a while since I've submitted PRs for MacPorts. Thanks for putting up with my oops!

Reply:
I had to squash and force push so this will lose file version references, but I made the python.versions match.